src/NServiceBus.Callbacks/CallbacksSupport.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NServiceBus.Features
+{
+    using System;
+    using NServiceBus.Callbacks;
+
+    class CallbacksSupport : Feature
```

Comment:
Are u nitpicking ?
It is an internal class, not really following you naming strategy ?